### DISCUSSION TITLE
Tag results for multi-instance site

### DISCUSSION THREAD

**Discussion by andrewnicols** [facebook/docusaurus#7077] [Q&A] [answered]
If we have a multi-instance setup of the docs plugin, is it possible to comine all results matching a certain tag?

Some of our documentation is versioned, and other parts not (i.e. Mission statement, Roadmap, Coding conventions, etc.). We want to apply the same front matter tags across all of the documentation, but if I follow the tag link to see related content, it only shows content for that specific instance of the docs plugin.

Is there any way to show tag results for all instances of the docs plugin, rather than just within one specific instance?

Thanks

**Comment by Josh-Cena** [comment, marked as answer]
No, plugin instances are sandboxed, so currently a plugin has no way to know the tags created by another instance. I'm also not aware of any non-trivial workaround, since tags are not global data so you can't create a custom page to display them ㄟ(▔,▔)ㄏ

**Reply by andrewnicols** [reply]
Is this something that would be desirable in the longer term or something that Docusaurus may offer in the future?

**Reply by Josh-Cena** [reply]
An API that works out-of-the-box would be hard to design, so mostly we'll power you to build this in userland with #6923

**Reply by andrewnicols** [reply]
Thanks. That seems like a sensible way to go if it lands any time soon.

**Reply by slorber** [reply]
> Is this something that would be desirable in the longer term or something that Docusaurus may offer in the future?

That seems unlikely

Also keep in mind that tags pages are versioned. If you are on v2.0 doc, you don't see v1.0 tags, and /v2/my-tag only display v2 tagged docs.

Now if you have `/global/tags/my-tag`, should it show docs from `/versioned/1.0/tags/my-tag` or `/versioned/2.0/tags/my-tag`? 🤷‍♂️ there's no good way for Docusaurus to understand what you are trying to do here
